Tests/DatadogIntegrationTests/UITestsHelpers.swift

```diff
@@ -78,3 +78,14 @@ extension String {
 struct Exception: Error, CustomStringConvertible {
     let description: String
 }
+
+extension XCUIElement {
+    @discardableResult
+    func safeTap(within timeout: TimeInterval = 0) -> Bool {
+        if waitForExistence(timeout: timeout) && isHittable {
+            tap()
+            return true
+        }
+        return false
```

Comment:
I don't see how can we use these return values in our UITests - as we're not using them now, let's keep it simple:
```swift
func safeTap(within timeout: TimeInterval = 0)
```
💭?